### DISCUSSION TITLE
Diffing mechanism to improve build times?

### DISCUSSION THREAD

**Discussion by kawpii** [vercel/turborepo#601] [Ideas]
Have anyone considered using a [diffing mechanism like in Nx](https://github.com/vsavkin/large-monorepo#why-is-nx-faster-than-turbo) to improve build times?

**Comment by jaredpalmer** [comment]
Yes, it's in the works!

**Reply by kawpii** [reply]
Glad to know!

**Reply by kawpii** [reply]
@jaredpalmer How is the progress with the diffing in Turborepo?

**Reply by nathanhammond** [reply]
We'll be handling it from the daemon that is coming in 1269! Slow, incremental progress.
